resources/views/product/overview.blade.php

```diff
@@ -20,7 +20,7 @@
                     @includeWhen($product->type_id !== 'grouped', 'rapidez::product.partials.addtocart')
                     @if (App::providerIsLoaded('Rapidez\Reviews\ReviewsServiceProvider'))
                         @if ($product->reviews_score)
-                            <stars :score="{{ $product->reviews_score }}" :count="{{ $product->reviews_count }}"></stars>
+                            <x-rapidez-reviews::stars score="{{ $product->reviews_score }}" count="{{ $product->reviews_count }}" />
```

Comment:
```suggestion
                            <x-rapidez-reviews::stars :score="$product->reviews_score" :count="$product->reviews_count" />
```

This also works right?